### PR TITLE
Add event Phase::Initialization

### PR DESCRIPTION
**Pull request by h4x3rotab**
This is introduced since https://github.com/paritytech/substrate/commit/2c87fe171bc341755a43a3b32d67560469f8daac

https://github.com/paritytech/substrate/blob/fa404167bbfe7bfd459689d25fc19b86da698367/frame/system/src/lib.rs#L278-L285

### CODE REVIEW

**Review by niklasad1 — APPROVED** [review]

**Comment by h4x3rotab** [comment]
The build is failed due to an upstream compiling issue https://github.com/paritytech/substrate/issues/7844

**Review by emostov — APPROVED** [review]

**Comment by h4x3rotab** [comment]
Could we rerun the CI?

**Comment by niklasad1** [comment]
Funny thing, we run into https://github.com/libp2p/rust-libp2p/pull/1912 now :(

**Comment by emostov** [comment]
@niklasad1 would bumping to `parity-multiaddr-0.10.1` fix this (the version that was published 6 hours ago)?

**Comment by niklasad1** [comment]
> would bumping to parity-multiaddr-0.10.1 fix this (the version that was published 6 hours ago)?

Yes if it would be a direct dependency but unfortunately, in this case, it's fetched via [sc-network](https://crates.io/crates/sc-network) which depends on an old version of [libp2p](https://crates.io/crates/libp2p) so we are blocked until a new release :(

Still, I think it's fine the merge this PR anyway

**Comment by emostov** [comment]
> Still, I think it's fine the merge this PR anyway

@niklasad1 (@dvdplm) who makes the final call on this?

**Comment by niklasad1** [comment]
I think it is up to @ascjones :)

However, the fix is being backported to [parity-multiaddr 0.9](https://github.com/libp2p/rust-libp2p/pull/1925) so let's wait for it,

**Review by ascjones — APPROVED** [review]

**Pull request merged**
